bin/setup.js

```diff
@@ -15,7 +15,15 @@ if (saveRequested) {
   var packageJson = require(path.join(pathToAppRoot, 'package.json'))
 
   packageJson.scripts = packageJson.scripts || {}
-  packageJson.scripts[ 'start' ] = 'hoodie'
+
+  if (packageJson.scripts[ 'start' ]) {
+    log.info('setup', 'start script already set to "' + packageJson.scripts[ 'start' ] +
+    ', you can start hoodie with "npm run start-hoodie" instead')
+
+    packageJson.scripts[ 'start-hoodie' ] = packageJson.scripts[ 'start' ]
```

Comment:
@gr2m @Charlotteis Now I fixed wrong named variables. And do I need to change line 23 to `packageJson.scripts[ 'start-hoodie' ] = 'hoodie'`? I'm not sure I should change it.